spec/spec_helper.rb

```diff
@@ -188,12 +188,6 @@ def setup_runtime_log
 
   config.raise_errors_for_deprecations!
 
-  # sometimes stuff hangs -> do not hang everything
-  config.include(Module.new {def test_timeout;30;end })
```

Comment:
still need this though ?
... having tests just stop and hang is hard to debug :(

Reply:
Sure I can re-add that! But I think the `Timeout` block was actually swallowing some errors and making it super hard to figure out this test failure, and when I removed it the test started failing consistently, so it was easy to fix. So my theory is that those regular timeout errors might have actually been caused by this `ordered` expectation.

But I'll run my delayed commit script again for a while and create 15 more builds, and will check to see if there's any other flaky tests.

Reply:
I remember when I last worked on stabilizing CI here, I also temporarily removed this because it seemed to be indeed swallowing errors somehow. :+1:

Reply:
bringing them back in https://github.com/grosser/parallel_tests/pull/741 with a dedicated class and note on debugging